include/TrustWalletCore/TWPrivateKey.h

```diff
@@ -60,6 +60,11 @@ struct TWPublicKey *_Nonnull TWPrivateKeyGetPublicKeyEd25519Extended(struct TWPr
 TW_EXPORT_METHOD
 struct TWPublicKey *_Nonnull TWPrivateKeyGetPublicKeyCurve25519(struct TWPrivateKey *_Nonnull pk);
 
+/// Computes an EC Diffie-Hellman secret in constant time
+/// Supported curves: secp256k1
+TW_EXPORT_METHOD
+TWData *_Nullable TWPrivateKeyGetSharedKey(struct TWPrivateKey *_Nonnull pk, struct TWPublicKey *_Nonnull publicKey, enum TWCurve curve);
```

Comment:
Add consts to const input parameters

Reply:
@catenocrypt Are you sure about this? E.g. `sign` method below doesn't have this annotation in exposed wrapper. When I add it, it breaks iOS tests (`no symbol found for architecture` error)

Reply:
nevermind, seems like my mistake - forgot to add same qualifier to wrapper implementation